crates/oxc_semantic/src/builder.rs

```diff
@@ -467,8 +484,11 @@ impl<'a> Visit<'a> for SemanticBuilder<'a> {
         program.scope_id.set(Some(self.current_scope_id));
 
         /* cfg */
```

Comment:
If you can be bothered, I think all these `/* cfg */` comments could be removed, as the `cfg!` macro now performs same function of delimiting the code related to CFG. But this is just a style nit - feel free to ignore if you don't have time/don't agree.

Reply:
I thought of it but didn't do it since there are a handful of instances where we have cfg related code outside of this macro, Wanted to keep it as a convention for that reason, and I'd love to remove these when it is a bit more mature and we know all of our cfg logics can be contained in these macro scopes.